spec/helpers/script_helper_spec.rb

```diff
@@ -82,8 +82,8 @@
         render_javascript_pack_once_tags
 
         expect(response.headers['link']).to eq(
-          '</application.js>; rel=preload; as=script,' \
-            '</document-capture.js>; rel=preload; as=script',
+          '</application.js>;rel=preload;as=script,' \
+            '</document-capture.js>;rel=preload;as=script',
```

Comment:
Whitespace is optional per the spec, which allows us to salvage back some of the bytesize increase these changes could incur.

>`link-value = "<" URI-Reference ">" *( OWS ";" OWS link-param )`

https://httpwg.org/specs/rfc8288.html#header

(["OWS" being defined as "optional whitespace"](https://httpwg.org/specs/rfc8288.html#notational-conventions))